SU-HSDO/suhumsci
# fix(shs-5223): fix a11y contrast in headings on grouped tables view

# READY FOR REVIEW

## Summary
Add a new utility class to fix the caption color on grouped tables view

Need Review By (Date)
02/14

## Urgency
low

## Steps to Test
1. Go this page https://economics.suhumsci.loc/graduate/student-placement
2. Edit the view
3. Click on `Content: Graduation Year [hidden]`
4. Update the `Formatter` to be `Custom` and the `Date/time format` to be just `Y`
5. Scroll down to the `Rewrite results` and update it to be `<div class="text-align-center hb-heading-2">{{ field_hs_person_student_grad }}</div>`. I added a new utility class (`hb-views-table-caption`, to fix the constrast issue)
6. Save and confirm the caption/heading of the table looks good and that Editorially is not flagging anything as an error

<img width="1050" alt="Screenshot 2024-02-01 at 4 52 59 PM" src="https://github.com/SU-HSDO/suhumsci/assets/48533432/4eef3fcc-8f12-4525-be04-fdbbd6ef57fa">


7. Now, go to https://economics.suhumsci.loc/undergraduate/honors-program/honors-prize-and-award-winners
8. Edit the view
9. Click on `Content: Date [hidden]`
10. Scroll down to the `Rewrite results` and update it to be `<div class="text-align-center hb-heading-2">{{ custm_hs_thesis_date }}</div>`
11. Save and confirm the caption/heading of the table looks good and that Editorially is not flagging anything as an error

<img width="1035" alt="Screenshot 2024-02-01 at 4 52 00 PM" src="https://github.com/SU-HSDO/suhumsci/assets/48533432/c0af96de-a58a-4c8f-8e13-95c8b917809c">


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
